python/ray/train/tests/test_backend.py

```diff
@@ -405,6 +405,49 @@ def test():
             assert worker_result != placement_group.id
 
 
+def test_auto_transfer_data_from_host_to_device():
+    import torch
+    import numpy as np
+
+    ray.init(num_cpus=1, num_gpus=1)
```

Comment:
Create a new pytest fixture for this? Similar to `ray_start_4_cpus_2_gpus` in `test_gpu`.